uhubctl.c

```diff
@@ -157,6 +159,7 @@ int print_usage()
         "--vendor,   -n - limit hub by vendor id [%s] (partial ok).\n"
         "--delay,    -d - delay for cycle action [%d sec].\n"
         "--repeat,   -r - repeat power off count [%d] (some devices need it to turn off).\n"
+        "--reset,    -R - reset hub after each action.\n"
```

Comment:
Oops, I meant to change this to say only after each power-on action, now it might be misleading.